docs/core/windows-prerequisites.md

```diff
@@ -11,7 +11,7 @@ ms.devlang: dotnet
 ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
 ---
 
-# Prerequisites for Windows development
+# Prerequisites for Windows development and/or running .Net Core applications
```

Comment:
.NET needs to be all caps

How about we change the title to something like "Prerequisites for .NET Core on Windows"?